src/components/common/modal/SubscriptionModalContent.tsx

```
@@ -1,51 +1,77 @@
 import { verifyEmail } from "@/hooks/api/member/useApiVerifyEmail.ts";
 import { sendSubscribeEmail } from "@/hooks/api/subscribe/useApiSendSubscribe";
-import { BREAK_POINTS, DESIGN_SYSTEM_COLOR } from "@/style/variable";
 import { EMAIL_PURPOSE } from "@/utils/common";
 import { useModal } from "@/hooks/useModal";
-import { EMAIL_REGEX } from "@/utils/regex";
+import Input from "../Input";
+import CertificateBox from "@/components/app/signup/certificate-box";
+import useCheckEmail from "@/hooks/useCheckEmail";
+import Button from "../Button";
+import Chip from "../chip";
+import { set } from "lodash";
```

ESLint

[error] 'set' is defined but never used.

(@typescript-eslint/no-unused-vars)